### PR TITLE
fixes in cookMealInfo

### PR DESCRIPTION
**Pull request by noorafi**
ref #66 : repull request for cook meal info name changes, removed console.logs.

### CODE REVIEW

**Comment by Karyum** [comment]
@noorafi can we change the title of the pull request to something better that reflects what this pull request is about ?

File: client/src/actions/getCookInfo.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+import { GET_COOK_MEAL_INFO } from "./actionTypes";
+
+const getCookInfo = mealId => {
```
Comment (DenisKent) [inline client/src/actions/getCookInfo.js]: The name of the action should also change

File: client/src/actions/getCookInfo.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  return dispatch => {
+    dispatch({
+      type: GET_COOK_MEAL_INFO,
+      payload: axios.get(`/api/mealCookInfo/${mealId}`)
+    });
+  };
+};
+
+export default getCookInfo;
```
Comment (DenisKent) [inline client/src/actions/getCookInfo.js]: Name change

File: client/src/actions/getCookInfo.js (hunk 1 above)
Comment (DenisKent) [reply client/src/actions/getCookInfo.js]: The name of this file should also change

File: client/src/pages/MealPage/MealPage.js [hunk 3]
```diff
@@ -2,13 +2,15 @@ import React from "react";
...
 import MealInfo from "./sections/MealInfo/MealInfo";
+import CookInfo from "./sections/CookInfo/CookInfo";
 import MealReviews from "./sections/MealReviews/MealReviews";
 
 const MealPage = props => {
   return (
     <React.Fragment>
       <MealInfo mealId={props.match.params.mealId} />
-      <MealReviews mealId={props.match.params.mealId}/>
+      <CookInfo cookMealId={props.match.params.mealId} />
```
Comment (DenisKent) [inline client/src/pages/MealPage/MealPage.js]: the name of this component

File: client/src/pages/MealPage/sections/CookInfo/CookInfo.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
```
Comment (DenisKent) [inline client/src/pages/MealPage/sections/CookInfo/CookInfo.js]: The name of this file

File: client/src/reducers/index.js [hunk 5]
```diff
@@ -2,12 +2,14 @@ import { combineReducers } from "redux";
...
 import getMealListReducer from "./getMealListReducer";
+import getCookMealInfoReducer from "./getCookMealInfoReducer";
 
 export default combineReducers({
   addToBasket: addBasketReducer,
   mealData: getMealInfoReducer,
+  mealList: getMealListReducer,
   mealReviews: getMealReviewsReducer,
-  mealList: getMealListReducer
+  cookData: getCookMealInfoReducer
```
Comment (DenisKent) [inline client/src/reducers/index.js]: name of this object key

File: server/controllers/cookInfo.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+const { getCookInfo } = require('../database/getQueries/getCookInfo');
```
Comment (DenisKent) [inline server/controllers/cookInfo.js]: name

File: server/controllers/index.js [hunk 7]
```diff
@@ -7,8 +7,11 @@ const mealInfo = require('./mealInfo');
 const mealReviews = require('./mealReviews');
 const mealList = require('./mealList');
 
+const mealCookInfo = require('./cookInfo');
```
Comment (DenisKent) [inline server/controllers/index.js]: ./cookInfo name

**Comment by noorafi** [comment]
@DenisKent, me and suha changed them if you still dont like the names tell us what the names youd like them to be and we'll change them. 
\

**Comment by matthall8** [comment]
@noorafi @afifisuha - I've made the name changes as you guys moved onto the login pages.

**Comment by noorafi** [comment]
@matthall8 thanks man :+1:

**Comment by matthall8** [comment]
Thanks @jema28.. @Karyum over to you.

**Pull request merged**